core/operation/src/main/java/uk/gov/gchq/gaffer/operation/impl/delete/DeleteElements.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2024 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.operation.impl.delete;
+
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+import uk.gov.gchq.gaffer.commonutil.ToStringBuilder;
+import uk.gov.gchq.gaffer.data.element.Element;
+import uk.gov.gchq.gaffer.operation.Operation;
+import uk.gov.gchq.gaffer.operation.Validatable;
+import uk.gov.gchq.gaffer.operation.io.MultiInput;
+import uk.gov.gchq.koryphe.Since;
+import uk.gov.gchq.koryphe.Summary;
+
+import java.util.Map;
+
+/**
+ * A {@code DeleteElements} operation is a {@link Validatable} operation
+ * for deleting elements. This is a core operation that all stores should
+ * be able to handle.
+ *
+ * This operation requires an {@link Iterable} of {@link Element}s to be deleted.
+ * Handlers should throw an {@link uk.gov.gchq.gaffer.operation.OperationException}
+ * if unsuccessful.
+ *
+ * For normal operation handlers the operation
+ * {@link uk.gov.gchq.gaffer.data.elementdefinition.view.View} will be ignored.
+ *
+ * @see DeleteElements.Builder
+ */
+@JsonPropertyOrder(value = { "class", "elements" }, alphabetic = true)
+@Since("2.2.2")
```

Comment:
?

Reply:
Needs bumping to 2.2.3 or 2.3.0

Reply:
Yeah think can go for 2.3.0 for next release